terraform/install.sh

```diff
@@ -150,15 +215,15 @@ installTerraform
 #export GOOGLE_APPLICATION_CREDENTIALS=""
 #gcloud auth application-default login
 
-# Ensure no google.com accounts early - they are not supported!
 acct=$(gcloud info --format="value(config.account)")
 if [[ $acct == *"google.com"* ]];
 then
-  log "Google.com accounts are currently not supported by Stackdriver Sandbox.";
-  exit;
+  folder_id="262044416022"            
```

Comment:
We should also add instructions for Googlers:  
we should show them a prompt that asks them to apply for permissions (one off) to create projects under /experimental folder. In the prompt they need to acknowledge that they applied or cancel. If they acknowledged that they applied, we run the rest of the flow that creates the project under /experimental for google.com accounts.

Reply:
Ok.